data/ZTFrlh6cyjh_phot.csv

```diff
@@ -1,1122 +1,1122 @@
-ztfi,58340.361829999834,29322.432823031155,25.0,ab,297.0765468198838,0.25,0.22,342.459423,51.2634152
-ztfi,58346.34618999995,19072.16520593907,25.0,ab,193.22724734178502,0.25,0.22,342.4595477,51.2634045
-ztfi,58763.19981000014,29322.432823031155,25.0,ab,297.0765468198838,0.25,0.22,342.4596214,51.2633781
-ztfi,58761.27215000009,31102.83668685142,25.0,ab,315.114485042836,0.25,0.22,342.45961,51.2634233
-ztfi,58732.3932500002,30647.849053700294,25.0,ab,310.5048349596368,0.25,0.22,342.4596475,51.2633845
-ztfi,58762.25511999987,30789.313097416132,25.0,ab,311.93806015823867,0.25,0.22,342.4596167,51.2634023
-ztfi,58758.24707000004,15703.628043335548,25.0,ab,159.09933598667064,0.25,0.22,342.459609,51.2633867
-ztfi,58803.18439000007,29895.09927814587,25.0,ab,302.87844511365154,0.25,0.22,342.45962,51.2633878
-ztfi,58760.346439999994,28340.010055090956,25.0,ab,287.12325388616546,0.25,0.22,342.4596308,51.2634136
-ztfi,58764.18502999982,31017.0149947769,25.0,ab,314.24499334418977,0.25,0.22,342.4596122,51.263381
-ztfi,58749.24249000009,30422.85727480961,25.0,ab,308.2253589987196,0.25,0.22,342.4596288,51.263397
-ztfi,58747.24142999994,27214.477464543666,25.0,ab,275.7200617516302,0.25,0.22,342.4596355,51.26339
-ztfi,58750.25253999978,31622.776601683792,25.0,ab,320.38219101336193,0.25,0.22,342.4596052,51.2633741
-ztfi,58748.24265999999,15588.344677043906,25.0,ab,157.93135703450153,0.25,0.22,342.4596338,51.263401
-ztfi,58747.200389999896,20044.720273651634,25.0,ab,203.08056690950156,0.25,0.22,342.4596237,51.2633956
-ztfi,58758.34806999983,30450.89066974223,25.0,ab,308.509375819987,0.25,0.22,342.4596293,51.2634048
-ztfi,58751.25231999997,34705.63540079883,25.0,ab,351.6157878946903,0.25,0.22,342.4596239,51.2633878
-ztfi,58751.24262000015,31651.915672262647,25.0,ab,320.6774098486254,0.25,0.22,342.4596194,51.2634101
-ztfi,58831.11551999999,30874.504755874485,25.0,ab,312.8011687504012,0.25,0.22,342.4596454,51.2634079
-ztfi,58761.25186000019,31506.488329723954,25.0,ab,319.2040309855778,0.25,0.22,342.4596252,51.2633849
-ztfi,58760.24319999991,31102.83668685142,25.0,ab,315.114485042836,0.25,0.22,342.4596298,51.2633971
-ztfi,58745.242999999784,31622.776601683792,25.0,ab,320.38219101336193,0.25,0.22,342.4596214,51.26338
-ztfi,58750.2014299999,31188.895840939353,25.0,ab,315.98638255805866,0.25,0.22,342.459654,51.2634081
-ztfi,58751.199080000166,31017.0149947769,25.0,ab,314.24499334418977,0.25,0.22,342.45962,51.263391
-ztfi,58749.20110000018,30874.504755874485,25.0,ab,312.8011687504012,0.25,0.22,342.4596203,51.2633911
-ztfi,58754.251600000076,30394.84968770113,25.0,ab,307.9416036461826,0.25,0.22,342.45963,51.2633985
-ztfi,58748.2661799998,14777.46706171993,25.0,ab,149.71605227763357,0.25,0.22,342.4596264,51.2633981
+filter,mjd,flux,zp,magsys,fluxerr,ra_unc,dec_unc,ra,dec,ref_flux,ref_fluxerr
+ztfg,58744.20135999983,30988.46042012061,25.0,ab,371.0385501207088,0.25,0.22,342.4596227,51.2633822,100.0,0.1
```

Comment:
I know it is a pain but would be nice to have a bigger ref flux to make it show up in the plot.

Reply:
I made it larger.